### DISCUSSION TITLE
［推销］绳宝，一款真正人性化的智能语音

### DISCUSSION THREAD

**Discussion by FarMounTAI** [share121/inter-knot#1012] [其他]
绳匠，你还在日日夜夜忙忙碌碌受理委托却迟迟找不到适合你的委托吗？绳匠，你还在真真切切任劳任怨对待客户却无法拿到你应有的报酬吗？绳宝，它来了。

绳宝是由绳网论坛研发及生产的，新艾利都首个内置语音包的智能机器人（水壶），绳宝专为新手绳匠而定制，是一款真正人性化的智能语音，更懂委托更懂你，内含58个学科，21门外语，模块化设计，核心兼容邦布语音模块改装，是邦布的贴心老师。更内置了黑手哥、虎哥、赵铁柱和骨灰扬诺夫语音包，宇宙最强人工智能-Fairy同款算法，不但能干扰以骸的以太心智活性，更能让您面对委托人的砍价时不再唯唯诺诺，忍气吞声，对线词汇不再单调，真正是您家庭的一员。法厄同亲选认证，保证脚踢Fairy，秒杀一切。

现在预定仅需1919810丁尼，更有青春版和长辈版供选购，114514丁尼起售。
![IMG_20240810_205327](https://github.com/user-attachments/assets/c9b3149c-d08b-4b59-9b8a-dc4c9d0edf0b)

**Comment by Kaohaaa** [comment]
太贵了吧，我得去北国银行借一点

**Comment by Qiling-XwX** [comment]
已下单，账单请送到卡卡罗团长手上

**Comment by xibozhi** [comment]
只要在乔普师傅那少吃400碗面就可以拿下（划掉）

**Comment by arkdecade** [comment]
已下单，账单请送到飞电或人先生手上

**Comment by KiryuEmu** [comment]
未下单，未付款，期待发货

**Comment by uyfh352** [comment]
可以分期付款嗎，只有200丁尼

**Comment by Ropemakers5302** [comment]
有人买了吗？好不好用，我怕踩空
